cmd/kanikoExecute.go

```diff
@@ -175,7 +203,21 @@ func runKanikoExecute(config *kanikoExecuteOptions, telemetryData *telemetry.Cus
 					containerImageNameAndTag := fmt.Sprintf("%v:%v", config.ContainerImageName, containerImageTag)
 					commonPipelineEnvironment.container.imageNameTag = containerImageNameAndTag
 				}
-
+				if config.CreateBOM {
+					//Syft for multi image, generates bom-docker-(1/2/3).xml
+					shellRunner.AppendEnv([]string{"DOCKER_CONFIG", "/kaniko/.docker"})
```

Comment:
just wondering if we can refactor this to avoid code duplication.. at least we can put this `shellRunner.AppendEnv([]string{"DOCKER_CONFIG", "/kaniko/.docker"})` into `installSyft` method

Reply:
Good idea. 
But for installation of syft we don't need it and I feel it is a bit of polluting the method
Would it be okay if we leave it as it is for now?

Reply:
Good idea.
But installSyft downloads and installs syft and I feel this might pollute the purpose a bit.
Would it be okay if we leave it like this for now?